<?xml version="1.0"?>
<launch>
  <arg name="display" default="true"/>

  <arg name="ns"            default="stereo_pair"/>
  <arg name="tf_prefix"     default="$(arg ns)"/>

  <arg name="service"       default="False"/>

  <arg name="markers"       default="markers"/>

  <arg name="image"         default="left/image_rect_color"/>
  <arg name="camera_info"   default="left/rec_camera_info"/>
  <arg name="dictionary"    default="3"/>

  <!-- corners to be consistent with checker board - squares minus one -->
  <arg name="board_width"    default="5"/><!-- corners to be consistent with checker board -->
  <arg name="board_height"   default="3"/><!-- corners to be consistent with checker board -->
  <arg name="square_length"  default="65.0"/><!--mm-->
  <arg name="marker_length"  default="51.0"/><!--mm-->

  <include file="$(find aruco_detector)/launch/base_marker.launch" pass_all_args="true">
    <arg name="marker_type" value="charuco"/>
    <arg name="sensor_type" value="mono"/>
  </include>
</launch>
